### DISCUSSION TITLE
Egress causes a dns timeouts

### DISCUSSION THREAD

**Discussion by simonoff** [kube-hetzner/terraform-hcloud-kube-hetzner#830] [Q&A]
Hello guys,

First of all thank you for such amazing project!

I have fighting whole weekend with up and run cluster with egress support.
At the time i'm enabling egress than eveything stop to work. 

So there was a question - what was the original reason to set devices to `eth0` strictly in cilium?
Only after i have removed such option all starts working as expected.

**Comment by mysticaltech** [comment]
@simonoff For normal clusters it should be set to eth1 only, as we are only concerned with the private interface for the Hetzner private network which the cluster runs on. But for a setup with Egress, you need both, so you can remove that specification to have it consider both interfaces. However in none, it should be specified to eth0, if that is the case, please do open and issue or provide a PR to fix it. Thanks!

**Reply by simonoff** [reply]
Ok, i will later will create PR for docs. Right now I'm playing a bit with it.
